### DISCUSSION TITLE
NeoPixel_RGB with pushButton

### DISCUSSION THREAD

**Discussion by Nezaemmy** [HomeSpan/HomeSpan#926] [Q&A]
Hi
It is just a question, Is it possible to integrate NeoPixel_RGB with PushButton?
The PushButton will be to turn ON and OFF NeoPixel_RGB or double-click to change the color.
I tried, but I was not able to do it.

**Comment by HomeSpan** [comment]
(converted to discussion - please use Issues to report bugs).

Yes, you can combine buttons and pixels in the same sketch.  See the PushButton Examples for tutorials on how to use PushButtons and then the Pixel Examples for how to control a NeoPixel.  You can then start with the PushButton example and change the code to turn on/off a NeoPixel instead of an LED.  Alternatively, you can start with the Pixel example and change the code to add in a SpanButton.  Lots of different ways of accomplishing this.  As always, the Tutorials are the best way to start.
